searches/exponentialsearch.py

```diff
@@ -0,0 +1,80 @@
+# Author:- Arnab Nath
+# Python program to find an element x
+# in a sorted array using Exponential Search
+
+# A recursive binary search function returns
+# location of x in given array arr[l..r] is
+# present, otherwise -1
+arr = [4, 7, 10, 34, 44, 70]
+n = len(arr)
+x = 44
+if arr[0] == x:
+    print("0")
+i = 1
+# Finding range for binarySearch
+while i < n and arr[i] <= x:
+    i = i * 2
+min1 = min(i, len(arr))
+
+
+def binarysearch(arr: list, le: int, r: int, x: int) -> int:
```

Comment:
```suggestion
def binarysearch(arr: list, le: int, ridiculous_name: int, xtra_long_variable_name: int) -> int:
```
Please change these to something more self-documenting.